docs/reference/API/element.md

```diff
@@ -0,0 +1,295 @@
+# `Element`
+
+The `Element` API is a helpful way to create and manipulate elements in the DOM. It is a wrapper around the native DOM API, and is designed to be as intuitive as possible.
+
+## Methods and Properties
+
+| Property | Description                             |
+|----------|-----------------------------------------|
+| `element`   | Returns the element with the given ID. |
+| `id`        | Returns the element's ID.              |
+| `value`    | Returns the element's value.           |
+| `innerHtml` | Returns the element's inner HTML.      |
+
+
+
+| Method               | Description                            |
+|----------------------|--------------------------------------------------------------|
+| `write`                | Writes `value` to element and handles various mime types. `append` defaults to `False`, if set to true, it will create a child element.          |
```

Comment:
Maybe we should be specific here and say which mimes we're rendering?
I made a list with the correct ones:


| Method              | Inferred MIME type     |
|---------------------|------------------------|
| `__repr__`          | text/plain             |
| `_repr_html_`       | text/html              |
| `_repr_svg_`        | image/svg+xml          |
| `_repr_png_`        | image/png*             |
| `_repr_pdf_`        | application/pdf        |
| `_repr_jpeg_`       | image/jpeg*            |
| `_repr_json_`       | application/json       |
| `_repr_javascript_` | application/javascript*|
| `savefig`           | image/png              |
|                     |                        |

For the rest, even though it's written in there, it doesn't work.